lib/msal-common/src/request/AuthorizationUrlRequest.ts

```diff
@@ -32,6 +32,7 @@ import { BaseAuthRequest } from "./BaseAuthRequest";
  * - nonce                      - A value included in the request that is returned in the id token. A randomly generated unique value is typically used to mitigate replay attacks.
  */
 export type AuthorizationUrlRequest = BaseAuthRequest & {
+    authenticationScheme?: AuthenticationType,
```

Comment:
Should we call this `authenticationType` to match the name of the type and avoid confusion?

Reply:
changed to authenticationScheme to match other MSAL libs. I can change the name of the type as well.